DSharpPlus.Commands/Processors/BaseCommandProcessor/BaseCommandProcessor.cs

```diff
@@ -23,6 +23,11 @@ public abstract class BaseCommandProcessor<TEventArgs, TConverter, TConverterCon
     where TConverterContext : ConverterContext
     where TCommandContext : CommandContext
 {
+    /// <summary>
+    /// Serves as a sentinel type for attempted conversions.
+    /// </summary>
+    protected class ConverterSentinel;

```

Comment:
What is a sentinel?

Reply:
a value other code can watch out for and treat specially, in this case necessary because we pass a dictionary that can't neatly be sorted after filling it (simply using an otherwise unusable reference type is probably the easiest way to go about this; i debated marking it as `file`-accessible but didn't, on the off-chance that custom processors would need to touch this)

Reply:
Could we update the docs to better explain that

Reply:
i feel like looking "sentinel type" or something up isn't that tall of an ask: <https://en.wikipedia.org/wiki/Sentinel_value>

Reply:
I suppose not